svc/LocalStorageService.js

```diff
@@ -16,6 +16,18 @@ import store from 'store2';
  */
```

Comment:
I think another line or two here in the docs explaining that this service falls back silently to transient memory if localstorage is not available (thanks to default settings in the store2 lib) would be helpful to developers.

Reply:
done